x-pack/plugins/cases/server/connectors/cases/cases_connector_executor.ts

```diff
@@ -982,13 +982,20 @@ export class CasesConnectorExecutor {
     const bulkCreateAlertsRequest: BulkCreateAlertsReq[] = casesUnderAlertLimit.map(
       ({ theCase, alerts }) => ({
         caseId: theCase.id,
-        attachments: alerts.map((alert) => ({
-          type: AttachmentType.alert,
-          alertId: alert._id,
-          index: alert._index,
-          rule: { id: rule.id, name: rule.name },
-          owner: theCase.owner,
-        })),
+        attachments: [
```

Comment:
Instead of creating one attachment per alert, we create one attachment with all alerts included in the attachment. This will reduce the amount of SO created.